src/util/loc.ts

```diff
@@ -0,0 +1,178 @@
+/*!
+ * Copyright (c) 2015-2016, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import Bundles from '@okta/okta-i18n-bundles';
+
+const bundleNames = ['login', 'country', 'courage'];
```

Comment:
1. I don't think we can do it automatically. But it's just 100 lines of code we need from courage (for `loc` util)
2. Yes, courage is large. And we use only `loc`, which is a small part of courage

Reply:
Is there a way to generate this file from the existing courage source rather than copy/pasting it?

Reply:
At a minimum we should add a note at the top of this file indicating where this is copied from and that both should be kept in sync.

Reply:
Added comment.

We could import only 1 file from courage instead whole courage package: https://github.com/okta/okta-signin-widget/blob/master/packages/%40okta/courage-dist/esm/src/courage/util/StringUtil.js
But it will also import underscore and jquery which is unwanted